pkg/node/manager/manager_test.go

```diff
@@ -829,3 +840,50 @@ func (s *managerTestSuite) TestNode(c *check.C) {
 	// Needs to be the same as n2
 	c.Assert(n, checker.DeepEquals, *n1V2)
 }
+
+func TestNodeManagerEmitStatus(t *testing.T) {
```

Comment:
Hmm yeah I didn't know about require.Eventually, I was hesitant to add the stream but I wanted to avoid non determinism by just using `time.Sleep()`

I'm going to change this over to just use the require approach, should clean things up quite a bit